src/poly.rs

```diff
@@ -670,17 +1014,19 @@ mod tests {
         // The whole first column never gets added up in practice, because nobody has all the
         // information. We do it anyway here; entry `0` is the secret key that is not known to
         // anyone, neither a dealer, nor a node:
-        let mut sec_key_set = Poly::zero();
+        let mut sec_key_set = Poly::zero().expect("Failed to create empty `Poly`");
         for bi_poly in &bi_polys {
-            sec_key_set += bi_poly.row(0);
+            sec_key_set += bi_poly.row(0).unwrap();
```

Comment:
Any reason this isn't an `expect` (not that it really matters)?

All of the other expect/panic messages look great by the way :)

Reply:
Whoops, missed one. Thanks